java/ql/lib/ext/com.fasterxml.jackson.databind.model.yml

```
@@ -5,6 +5,8 @@ extensions:
     data:
       - ["com.fasterxml.jackson.databind", "ObjectMapper", True, "convertValue", "", "", "Argument[0]", "ReturnValue", "taint", "manual"]
       - ["com.fasterxml.jackson.databind", "ObjectMapper", False, "createParser", "", "", "Argument[0]", "ReturnValue", "taint", "manual"]
+      - ["com.fasterxml.jackson.databind", "ObjectMapper", True, "readTree", "(URL)", "", "ReturnValue", ReturnValue, "taint", "ai-manual"] # result is remote, but only user-controlled if the URL is
+      - ["com.fasterxml.jackson.databind", "ObjectMapper", True, "readValue", "(InputStream,Class)", "", "ReturnValue", ReturnValue, "taint", "ai-manual"]
```

These models seem wrong. I think the first `ReturnValue` should be `Argument[0]`, and the second one lacks quotes.

```suggestion
      - ["com.fasterxml.jackson.databind", "ObjectMapper", True, "readTree", "(URL)", "", "Argument[0]", "ReturnValue", "taint", "ai-manual"] # result is remote, but only user-controlled if the URL is
      - ["com.fasterxml.jackson.databind", "ObjectMapper", True, "readValue", "(InputStream,Class)", "", "Argument[0]", "ReturnValue", "taint", "ai-manual"]
```

---

omg, sorry for that blunder (it's new code 🙈)